src/app/sitemap.ts

```diff
@@ -0,0 +1,19 @@
+import { CATEGORIES, SITE_URL } from '@/constants';
+import { MetadataRoute } from 'next';
+
+const createPages = (urls: string[]) => {
+    return urls.map((url) => ({
+        url: `${SITE_URL}${url}`,
```

Comment:
```suggestion
        url: `${SITE_URL}$(url.startsWith('/')? '' : '/'}${url}`,
```

Reply:
Fixed this per our conversation in slack. using get props.